docs/source/users/index.md

```diff
@@ -177,6 +177,43 @@ To compose a message, type it in the text box at the bottom of the chat interfac
     alt='Screen shot of an example "Hello world" message sent to Jupyternaut, who responds with "Hello world, how are you today?"'
     class="screenshot" />
 
+### Usage with SageMaker Endpoints
+
+Jupyter AI supports language models hosted on SageMaker Endpoints that use JSON
+APIs. The first step is to authenticate with AWS via the `boto3` SDK and have
+the credentials stored in the `default` profile.  Guidance on how to do this can
+be found in the
+[`boto3` documentation](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html).
+
+When selecting the SageMaker Endpoints provider in the settings panel, you will
+see the following interface:
+
+<img src="../_static/chat-sagemaker-endpoints.png"
+    height="50%"
+    width="50%"
```

Comment:
```suggestion
    style="transform: scale(0.5)"
```

Alternatively, this will reduce the X and Y dimensions by 50%, without having to update this every time you update the source image.

Reply:
Applying this change causes the image to be centered with an overly large margin. This is because the size of the image element is computed before the styles are applied.

<img width="841" alt="Screen Shot 2023-05-30 at 3 55 03 PM" src="https://github.com/jupyterlab/jupyter-ai/assets/44106031/507e1149-549e-4864-9b52-e51da3c98f72">